src/unchecked-uninit.md

```diff
@@ -64,6 +64,17 @@ This code proceeds in three steps:
 
 3. Finally, we have to change the type of our array to remove the
    `MaybeUninit`. With current stable Rust, this requires a `transmute`.
+   This transmute is legal because in memory, `MaybeUninit<T>` looks the same as `T`.
+
+    However, note that in general, `Container<MaybeUninit<T>>>` does *not* look
+   the same as `Container<T>`! Imagine if `Container` was `Option`, and `T` was
+   `bool`, then `Option<bool>` exploits that `bool` only has two valid values,
+   but `Option<MaybeUninit<bool>>` cannot do that because the `bool` does not
+   have to be initialized.
+
+    So, it depends on `Container` whether transmuting away the `MaybeUninit` is
+   allowed. For arrays, it is (and eventually the standard library will
+   acknowledge that by providing an appropriate methods).
```

Comment:
a single space indent is how you do this???? sdasfsdf

Reply:
Well, 4-space indent is how you do this. I used to then keep the entire paragraph 4-space indented, but this here IMO looks better when reading the ASCII.